EasyPost.Tests/InsuranceTest.cs

```diff
@@ -34,12 +34,7 @@ public async Task TestAll()
         {
             UseVCR("all");
 
-            InsuranceCollection insuranceCollection = await Client.Insurance.All(new Dictionary<string, object>
-            {
-                {
-                    "page_size", Fixture.PageSize
-                }
-            });
+            InsuranceCollection insuranceCollection = await Client.Insurance.All(new Dictionary<string, object> { { "page_size", Fixture.PageSize } });
```

Comment:
Question: So this is where I was wondering if we could get the best of both worlds where nested dicts break out and indent but individual lines don't:
```suggestion
            InsuranceCollection insuranceCollection = await Client.Insurance.All(new Dictionary<string, object> {
                { "page_size", Fixture.PageSize }
            });
```

Reply:
Correct:
https://www.jetbrains.com/help/resharper/EditorConfig_CSHARP_LineBreaksPageSchema.html#resharper_csharp_place_simple_initializer_on_single_line

Unfortunately, if there's only one item in the dictionary/list, it won't "chop" to a new line (note that setting 0 and 1 are the same): https://www.jetbrains.com/help/resharper/EditorConfig_CSHARP_LineBreaksPageSchema.html#resharper_csharp_max_initializer_elements_on_line

Reply:
Ah ok, I see some other examples where multiple line items do get broken up. I can live with that.

Reply:
Can I get a stamp?

Reply:
I... did?